api/api.proto

```diff
@@ -2684,3 +2698,53 @@ message LocateResponse {
     // <containerid>: /var/www
     map<string, string> dockerids = 2;
 }
+
+message VolumePlacementSpec {
+  // Defines a list of rules as part of the placement spec
```

Comment:
Does that mean that every rule must successfully be applied? Should add comments here on what happens if many rules are applied and if one conflicts with others.

Reply:
Added comment under `message VolumePlacementStrategy`.